src/main/java/com/fcgl/Listing/Vendors/AmazonProduct.java

```diff
@@ -0,0 +1,38 @@
+package com.fcgl.Listing.Vendors;
+
+import java.util.Objects;
+
+public class AmazonProduct {
```

Comment:
Need it to implement an interface because I'm doing a factory pattern for rabbit